### PR TITLE
CLI changes and configured

### PR DESCRIPTION
**Pull request by Garima03t**

### CODE REVIEW

File: BED/mulyankan/.vs/mulyankan/config/applicationhost.config
Comment (nisheesh) [inline BED/mulyankan/.vs/mulyankan/config/applicationhost.config:1]: Physical path may be different for all, do we need to add this in git ignore once we change in local or should it be part of commit?

File: BED/mulyankan/src/Project/mulyankan/bin/mulyankan.dll
Comment (nisheesh) [inline BED/mulyankan/src/Project/mulyankan/bin/mulyankan.dll:1]: dll needs to be ignored in commit

File: BED/mulyankan/src/Project/mulyankan/obj/Debug/mulyankan.csproj.FileListAbsolute.txt
Comment (nisheesh) [inline BED/mulyankan/src/Project/mulyankan/obj/Debug/mulyankan.csproj.FileListAbsolute.txt:1]: this also can vary among developers

File: BED/mulyankan/.vs/mulyankan/config/applicationhost.config
Comment (Garima03t) [reply BED/mulyankan/.vs/mulyankan/config/applicationhost.config:1]: @nisheesh correct this should be in .git ignore let me add tht

**Pull request merged**